seaclouds-catalog.bom

```diff
@@ -13,10 +13,11 @@ brooklyn.catalog:
         planner.url: https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=planner-service&v=LATEST&e=jar
         planner.config.port: 1234
         planner.config.discoverer.url: undefined
-        planner.config.optimizer.url: undefined
+        planner.config.optimizer.url: $brooklyn:formatString("http://%s:%s", component("optimizer-service").attributeWhenReady("host.address"), component("optimizer-service").attributeWhenReady("http.port"))
```

Comment:
@adriannieto Thank you. I see what you mean regarding the default value. I change it.